src/utils/middleware.js

```diff
@@ -1,9 +1,43 @@
+import Swagger from '@aeternity/aepp-sdk/es/utils/swagger';
+import { camelCase } from 'lodash-es';
 import { wrapTry } from './index';
 
-const backendFetch = (path, ...args) => wrapTry(
+const middlewareFetch = (path, ...args) => wrapTry(
   fetch(`${process.env.VUE_APP_MIDDLEWARE_URL}/${path}`, ...args).catch((err) => console.error(err)),
 );
 
-export default class Middleware {
-  static getAex9Balance = async (token, account) => backendFetch(`aex9/balance/${token}/${account}?top=true`);
+export default async function initMiddleware() {
```

Comment:
I think you can put this whole function to `initMiddleware` action because it will be used only there.